Fixes various wayland-related compile errors:

i965_output_wayland.c:31:41: fatal error: wayland-drm-client-protocol.h:
 No such file or directory

i965_output_wayland.h:31:26: error: unknown type name 'VADriverContextP'

make[4]: *** No rule to make target `wayland-drm-client-protocol.h', needed by
 `i965_output_wayland.c'.  Stop.

Patch sent upstream:
http://lists.freedesktop.org/archives/libva/2014-July/002520.html

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

diff -uNr libva-intel-driver-1.3.2.org/src/i965_output_wayland.c libva-intel-driver-1.3.2/src/i965_output_wayland.c
--- libva-intel-driver-1.3.2.org/src/i965_output_wayland.c	2014-06-03 02:36:44.000000000 +0200
+++ libva-intel-driver-1.3.2/src/i965_output_wayland.c	2014-07-19 11:48:31.329043292 +0200
@@ -28,7 +28,7 @@
 #include <va/va_backend.h>
 #include <va/va_backend_wayland.h>
 #include <wayland-client.h>
-#include <wayland-drm-client-protocol.h>
+#include <va/wayland-drm-client-protocol.h>
 #include "intel_driver.h"
 #include "i965_output_wayland.h"
 #include "i965_drv_video.h"
diff -uNr libva-intel-driver-1.3.2.org/src/i965_output_wayland.h libva-intel-driver-1.3.2/src/i965_output_wayland.h
--- libva-intel-driver-1.3.2.org/src/i965_output_wayland.h	2014-05-19 10:19:39.000000000 +0200
+++ libva-intel-driver-1.3.2/src/i965_output_wayland.h	2014-07-19 11:48:09.201321881 +0200
@@ -26,6 +26,7 @@
 #define I965_OUTPUT_WAYLAND_H
 
 #include <stdbool.h>
+#include <va/va_backend.h>
 
 bool
 i965_output_wayland_init(VADriverContextP ctx);
diff -uNr libva-intel-driver-1.3.2.org/src/Makefile.am libva-intel-driver-1.3.2/src/Makefile.am
--- libva-intel-driver-1.3.2.org/src/Makefile.am	2014-06-16 04:13:12.000000000 +0200
+++ libva-intel-driver-1.3.2/src/Makefile.am	2014-07-19 11:53:02.885627916 +0200
@@ -146,7 +146,7 @@
 endif
 
 # Wayland protocol
-protocol_source_h = wayland-drm-client-protocol.h
+#protocol_source_h = wayland-drm-client-protocol.h
 i965_output_wayland.c: $(protocol_source_h)
 @wayland_scanner_rules@
 
